src/main.rs

```diff
@@ -136,6 +137,50 @@ pub fn start(mut os_input: Box<dyn OsApi>, opts: Opt) {
     );
     let maybe_layout = opts.layout.map(Layout::new);
 
+    let send_app_instructions_clone = send_app_instructions.clone();
```

Comment:
Can we move this into the `set_hook` block in a similar way to which we do in the other threads? Just for consistency.

Something like:
```rust
std::panic::set_hook({
    let send_app_instructions = send_app_instructions.clone();
    Box::new(move |info| {
    ...
    });
});
```